pkg/loop/telem.go

```diff
@@ -66,9 +67,16 @@ func SetupTracing(config TracingConfig) error {
 	ctx, cancel := context.WithTimeout(ctx, 5*time.Second)
 	defer cancel()
 
+	var creds credentials.TransportCredentials
+	if config.TLSCertPath != "" {
+		creds, _ = credentials.NewClientTLSFromFile(config.TLSCertPath, "")
+	} else {
+		creds = insecure.NewCredentials()
```

Comment:
I'm definitely still thinking about this - I disallow setting `Tracing.TLSCertPath = ""` in the TOML configuration in core, so this path occurs when `Tracing.TLSCertPath` is unset. I'd like to be more verbal about this and log at the warn level. There's probably a sensible place to shout about this in core.

Reply:
Resolved by explicit insecure mode in TOML config